stripe-ui-core/src/test/java/com/stripe/android/uicore/elements/CountryConfigTest.kt

```diff
@@ -79,10 +80,10 @@ class CountryConfigTest {
 
         assertThat(
             defaultCountries.size
-        ).isEqualTo(249)
+        ).isEqualTo(235)
```

Comment:
Is the size change due to us not updating the list of unsupported countries?

Reply:
Definitely a behavior change! Before we default was the `Locale.getISOCountries`, and filtered by `supportedBillingCountries`. No it's always just `supportedBillingCountries`.